### PR TITLE
Add OpenTelemetry

### LINKED ISSUE #6838: Add OpenTelemetry icon
### Brand Name

OpenTelemetry

### Website

https://opentelemetry.io/

### Popularity Metric

OpenTelemetry is a CNCF project (https://www.cncf.io/blog/2021/08/26/opentelemetry-becomes-a-cncf-incubating-project/) and well-known in the observability space. It has gained a lot of momentum and adopted and contributed by major vendors and companies. https://github.com/open-telemetry has all the repos for SDKs for different languages plus the specification repo. 

### Official Resources for Icon and Color

https://cncf-branding.netlify.app/projects/opentelemetry/

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by joaopgrassi**
opentelemetry

**Issue:** Closes #6838

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Source from the CNCF project branding guidelines: https://cncf-branding.netlify.app/projects/opentelemetry/
- Used the `Icon black` SVG version
- Adapted the viewBox

### CODE REVIEW

**Comment by joaopgrassi** [comment]
@service-paradis here it is! I'm not sure about one thing though. 

The logo consists of two colors #f5a800 (Primary) and #425cc7 (Secondary). I took the black SVG logo as instructed in `CONTRIBUTING.md`. On `hex` what should I use? I can't  be only the primary as then the logo is wrong. Should I just use `000000` there then?

**Comment by joaopgrassi** [comment]
Also **before merging**: I want to get official confirmation that using this here is fine and we (me) are not infringing any trademark guidelines.

**Comment by service-paradis** [comment]
Hello @joaopgrassi, thank you for the contribution!

> The logo consists of two colors #f5a800 (Primary) and #425cc7 (Secondary) (from the guidelines). I took the black SVG logo as instructed in CONTRIBUTING.md. On hex what should I use? I can't use only the primary as then the logo is wrong (see preview below). I believe then 000000 is the most correct?

For the color, we really try to have the brand color as opposed to the logo color. In this case, the primary color (`#f5a800`) defined in their [guidelines](https://cncf-branding.netlify.app/projects/opentelemetry/) would be the correct one to use.

The preview colors are just there to display an example of what the icon would look like if colored using the hex. But Simple icons, as default, only provides icon without any color.

> Also before merging: I want to get official confirmation that using this here is fine and we (me) are not infringing any trademark guidelines.

I did not find any mention that would prevent the icon from being added to the project. Did you find something?

**Comment by joaopgrassi** [comment]
Hey @service-paradis 

> The preview colors are just there to display an example of what the icon would look like if colored using the hex. But Simple icons, as default, only provides icon without any color.

I see. But is there any possibility of it being used with the `hex` color there? Because if that's possible, it would be infringing the branding guidelines, thus also the trademarks. If not then I can add the primary color, even though that alone is "incorrect".

It would be nice if it would be possible to include more than one color for cases like this, where the logo is not based on a single color. Maybe this could be considered in the future.

> I did not find any mention that would prevent the icon from being added to the project. Did you find something?
I only skimmed through the text here: https://www.linuxfoundation.org/trademark-usage/. It also seems OK to me, but I don't want to run into legal issues 😅. They say in that page:

> [...] or to obtain written permission for a proposed use of The Linux Foundation trademarks, contact: trademarks@linuxfoundation.org

So I guess it doesn't hurt to send and be 100% safe then.

**Comment by service-paradis** [comment]
> I see. But is there any possibility of it being used with the hex color there? Because if that's possible, it would be infringing the branding guidelines, thus also the trademarks. If not, then I can add the primary color, even though that alone is "incorrect".

Yes, it would be possible for a user to use the icon with any color if they want. At this point, it is not our responsibility anymore. The same user could also have downloaded the SVG from their website to use it with any color. Users are responsible of respecting icon license and trademark when they use it. Since we provide a black version of their icon, we do respect their guidelines.

**Review by dirien — CHANGES_REQUESTED** [review]
HI @joaopgrassi,

thanks for contribution. There is one little difference with the original svg. Due to a reduction of points at the circle.

image

**Comment by joaopgrassi** [comment]
Hi @dirien! Oh, that's a good catch. I have to be honest here that I'm really not good with this stuff :sweat_smile:. I used Inkscape to edit the original svg but I was really lost. It would be great if someone could help :)

**Comment by dirien** [comment]
hi @joaopgrassi, any updates?

**Comment by joaopgrassi** [comment]
Hi @dirien! No updates from the Linux Foundation.. not sure if we should wait or just go ahead. About your comment the icon has a difference, I'm not sure if I'm capable of fixing it 😅. As I said this is all new and I'm not really good at it. If someone can help me or point on some resources.. then I can try.

**Comment by sachinraja** [comment]
Looks like it just needed a higher precision, this path matches perfectly:
```html
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 24 24"><title>OpenTelemetry</title><path d="M12.6974 13.1173c-1.0224 1.0224-1.0224 2.68 0 3.7024 1.0224 1.0224 2.68 1.0224 3.7024 0 1.0224-1.0223 1.0224-2.68 0-3.7024-1.0223-1.0223-2.68-1.0223-3.7024 0zm2.7677 2.7701c-.5063.5063-1.3267.5063-1.833 0s-.5063-1.3266 0-1.833c.5063-.5062 1.3267-.5062 1.833 0 .5063.504.5063 1.3267 0 1.833zM16.356.2355l-1.6041 1.6042c-.314.314-.314.83 0 1.144L21.015 9.247c.314.314.83.314 1.144 0l1.6042-1.6041c.314-.314.314-.83 0-1.144L17.4976.2354c-.314-.314-.8276-.314-1.1416 0zM5.1173 20.734c.2848-.2848.2848-.7497 0-1.0345l-.8155-.8155c-.2848-.2848-.7497-.2848-1.0345 0l-1.6845 1.6845-.0024.0024-.4625-.4625c-.2556-.2556-.6718-.2556-.925 0-.2556.2556-.2556.6718 0 .925l2.775 2.775c.2556.2556.6718.2556.925 0 .2532-.2556.2556-.6718 0-.925l-.4625-.4625.0024-.0024zm8.4856-15.893-3.5637 3.5637c-.3164.3164-.3164.8374 0 1.1538l2.2006 2.2005c1.5554-1.1197 3.7365-.981 5.1361.4187l1.7819-1.7818c.3164-.3165.3164-.8374 0-1.1538l-4.401-4.401c-.3165-.319-.8374-.319-1.1539 0zm-2.2881 7.8455-1.2999-1.2999c-.3043-.3043-.8033-.3043-1.1076 0l-4.5836 4.586c-.3042.3043-.3042.8033 0 1.1076l2.5973 2.5973c.3043.3043.8033.3043 1.1076 0l2.9478-2.9527c-.6231-1.2877-.5112-2.8431.3384-4.0383z"/></svg>
```

**Comment by joaopgrassi** [comment]
@sachinraja thanks for helping! Should I just take that version here? I'm not sure how can I check it against the original like  @dirien did so I'm accepting help. Otherwise I can just close this and wait for someone more competent to take over. 

Regarding the permission to use, I'm assuming we can go ahead since I never heard back from the Linux foundation. At least I have the email proofing I asked at least.

**Comment by dirien** [comment]
hi @joaopgrassi, you can c&p the one from @sachinraja and I will review it. Keep the PR open :)

**Comment by joaopgrassi** [comment]
@dirien cool! It's done. thanks @sachinraja

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6962,6 +6962,21 @@
...
+            "hex": "000000",
+            "source": "https://cncf-branding.netlify.app/projects/opentelemetry/",
+            "guidelines": "https://cncf-branding.netlify.app/projects/opentelemetry/",
+            "license": {
+                "type": "CC-BY-4.0",
+                "url": "https://www.linuxfoundation.org/trademark-usage/"
+            },
+            "aliases": {
+                "aka": [
+                    "otel"
```
Comment (LitoMore) [inline _data/simple-icons.json:6976]: Is this alias necessary? I can't find any OpenTelemetry related results in http://www.google.com/search?q=otel

Comment (joaopgrassi) [reply _data/simple-icons.json:6976]: Yes, it's well known in the community: https://github.com/open-telemetry/opentelemetry-specification#acronym

Comment (joaopgrassi) [reply _data/simple-icons.json:6976]: Good catch though, as I should have used `OTel`. I'll change it.

File: icons/opentelemetry.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...penTelemetry</title><path d="M12.6974 13.1173c-1.0224 1.0224-1.0224 2.68 0 3.7024 1.0224 1.0224 2.68 1.0224 3.7024 0 1.0224-1.0223 1.0224-2.68 0-3.7024-1.0223-1.0223-2.68-1.0223-3.7024 0zm2.7677 2.7701c-.5063.5063-1.3267.5063-1.833 0s-.5063-1.3266 0-1.833c.5063-.5062 1.3267-.5062 1.833 0 .5063.504.5063 1.3267 0 1.833zM16.356.2355l-1.6041 1.6042c-.314.314-.314.83 0 1.144L21.015 9.247c.314.314.83.314 1.144 0l1.6042-1.6041c.314-.314.314-.83 0-1.144L17.4976.2354c-.314-.314-.8276-.314-1.1416 0zM5.1173 20.734c.2848-.2848.2848-.7497 0-1.0345l-.8155-.8155c-.2848-.2848-.7497-.2848-1.0345 0l-1.6845 1.6845-.0024.0024-.4625-.4625c-.2556-.2556-.6718-.2556-.925 0-.2556.2556-.2556.6718 0 .925l2.775 2.775c.2556.2556.6718.2556.925 0 .2532-.2556.2556-.6718 0-.925l-.4625-.4625.0024-.0024zm8.4856-15.893-3.5637 3.5637c-.3164.3164-.3164.8374 0 1.1538l2.2006 2.2005c1.5554-1.1197 3.7365-.981 5.1361.4187l1.7819-1.7818c.3164-.3165.3164-.8374 0-1.1538l-4.401-4.401c-.3165-.319-.8374-.319-1.1539 0zm-2.2881 7.8455-1.2999-1.2999c-.3043-.3043-.8033-.3043-1.1076 0l-4.5836 4.586c-.3042.3043-.3042.8033 0 1.1076l2.5973 2.5973c.3043.3043.8033.3043 1.1076 0l2.9478-2.9527c-.6231-1.2877-.5112-2.8431.3384-4.0383z"/></svg>
```
Comment (sachinraja) [inline icons/opentelemetry.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>OpenTelemetry</title><path d="M12.6974 13.1173c-1.0224 1.0224-1.0224 2.68 0 3.7024 1.0224 1.0224 2.68 1.0224 3.7024 0 1.0224-1.0223 1.0224-2.68 0-3.7024-1.0223-1.0223-2.68-1.0223-3.7024 0zm2.7677 2.7701c-.5063.5063-1.3267.5063-1.833 0s-.5063-1.3266 0-1.833c.5063-.5062 1.3267-.5062 1.833 0 .5063.504.5063 1.3267 0 1.833zM16.356.2355l-1.6041 1.6042c-.314.314-.314.83 0 1.144L21.015 9.247c.314.314.83.314 1.144 0l1.6042-1.6041c.314-.314.314-.83 0-1.144L17.4976.2354c-.314-.314-.8276-.314-1.1416 0zM5.1173 20.734c.2848-.2848.2848-.7497 0-1.0345l-.8155-.8155c-.2848-.2848-.7497-.2848-1.0345 0l-1.6845 1.6845-.0024.0024-.4625-.4625c-.2556-.2556-.6718-.2556-.925 0-.2556.2556-.2556.6718 0 .925l2.775 2.775c.2556.2556.6718.2556.925 0 .2532-.2556.2556-.6718 0-.925l-.4625-.4625.0024-.0024zm8.4856-15.893-3.5637 3.5637c-.3164.3164-.3164.8374 0 1.1538l2.2006 2.2005c1.5554-1.1197 3.7365-.981 5.1361.4187l1.7819-1.7818c.3164-.3165.3164-.8374 0-1.1538l-4.401-4.401c-.3165-.319-.8374-.319-1.1539 0zm-2.2881 7.8455-1.2999-1.2999c-.3043-.3043-.8033-.3043-1.1076 0l-4.5836 4.586c-.3042.3043-.3042.8033 0 1.1076l2.5973 2.5973c.3043.3043.8033.3043 1.1076 0l2.9478-2.9527c-.6231-1.2877-.5112-2.8431.3384-4.0383z"/></svg>
```

**Review by sachinraja — CHANGES_REQUESTED** [review]

**Review by dirien — APPROVED** [review]
perfect
image

**Comment by joaopgrassi** [comment]
Thanks for having patience and to everyone that helped getting this merged! 🚀

**Pull request merged**